FunctionalPreprocessing/SubmitFunctionalPreprocessingJobSubmitHPC

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Same as for `SubmitFunctionalPreprocessingCheckHPC`, I think this belongs in a separate branch for merging later.